tools/config-markdown.py

```
@@ -1,207 +1,151 @@
+
+    # Header if exists
+    header_file = Path(__file__).parent / 'include' / 'markdown' / f"{item['id']}-header.md"
+    if header_file.is_file():
+        rel_path = f"tools/include/markdown/{item['id']}-header.md"
+        md.append(f"\n<!--- header START from {rel_path} --->")
+        md.append(header_file.read_text())
+        md.append(f"<!--- header STOP from {rel_path} --->\n")
+
+    # Commands
+    if item.get('command'):
+        for cmd in item['command']:
```

Ruff (0.8.2)

Loop control variable `cmd` not used within loop body

Rename unused `cmd` to `_cmd`

(B007)